### PR TITLE
Changed \ to / is "To start working on a new features" section

### PR DESCRIPTION
**Pull request by michellereneeg**
A forward slash results in a branch name such as "featuremich-readme," while the backslash actually does what we want: "feature/mich-readme."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ If you have any questions about this process, please use the internet as a resou
 - To start working on a new feature:
 
   - Make sure you are on the `develop` branch as this will have the latest changes to the codebase. You can run `git checkout develop` to switch to the branch if you are not already there.
-  - Create a new branch called `feature\<name>-<description>` by running the command `git branch feature\<name>-<description>`. Replace`<name>` with your name and `<description>` with a short description of the feature. For example, if Bongi was working on the README he might use the branch name `feature\bongi-readme`.
-  - Checkout (switch to) that branch by running `git checkout feature\<name>-<description>`.
-  - Make sure the remote repository is tracking your local branch. Run `git push -u origin feature\<name>-<description>`
+  - Create a new branch called `feature/<name>-<description>` by running the command `git branch feature/<name>-<description>`. Replace`<name>` with your name and `<description>` with a short description of the feature. For example, if Bongi was working on the README he might use the branch name `feature\bongi-readme`.
```
Comment (Mnumzane) [inline README.md]: Looks like you missed one change here.

Comment (Mnumzane) [reply README.md]: `feature\bongi-readme` also needs to be changed.

File: README.md [hunk 2]
```diff
@@ -54,8 +54,70 @@ If you have any questions about this process, please use the internet as a resou
...
+
+![Review required](https://user-images.githubusercontent.com/25873509/135943583-70c355ed-40b8-453e-8de1-2f6a2bb12a2e.png)
+
+Expanding the code, you can see the changes you commited. Code you added is highlighted in green. Code you deleted is highlighted in red. 
+
+You can also click the green "Review changes" button to add a comment.
+
+![Expanding the code](https://user-images.githubusercontent.com/25873509/135944141-f9e00b6e-f6ae-465a-9a3e-51eaa6798211.png)
+
+<u>**Note** that you **WILL NOT** be able to "Approve" or "Request changes" to your own pull request.</u>
```
Comment (Mnumzane) [inline README.md]: One way to highlight notes like this one with markdown is to use '>' before your important line.
> It ends up looking like this.

**Review by Mnumzane — CHANGES_REQUESTED** [review]
Looks really good so far! Thanks @michellereneeg 
See specific comments in code for 1 small change and 1 suggested change.

File: README.md (hunk 1 above)
Comment (michellereneeg) [reply README.md]: got it!

File: README.md (hunk 2 above)
Comment (michellereneeg) [reply README.md]: cool! i like it!

**Review by Mnumzane — APPROVED** [review]

**Pull request merged**